deployment/go.mod

```
@@ -521,7 +521,5 @@ require (
 replace (
 	// replicating the replace directive on cosmos SDK
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-	github.com/sourcegraph/sourcegraph => github.com/sourcegraph/sourcegraph-public-snapshot v0.0.0-20240822153003-c864f15af264
-
-	github.com/sourcegraph/sourcegraph/lib => github.com/sourcegraph/sourcegraph-public-snapshot/lib v0.0.0-20240822153003-c864f15af264
+	github.com/sourcegraph/sourcegraph/conc => github.com/sourcegraph/sourcegraph-public-snapshot/conc v0.0.0-20240822153003-c864f15af264
```

What is `/conc`?

---

I have no idea. It comes from //indirect.  I assume it's maybe something concurrency. We're not depending on it, go mod tidy adds it.

---

But it doesn't exist? https://github.com/sourcegraph/sourcegraph-public-snapshot/tree/c864f15af264f0f456a6d8a83290b5c940715349

---

https://github.com/sourcegraph/conc

---

I misread it. Fixing.

---

Oh well there is an extra `/sourcegraph` in here:
```suggestion
	github.com/sourcegraph/conc => github.com/sourcegraph/sourcegraph-public-snapshot/conc v0.0.0-20240822153003-c864f15af264
```

---

But that repo still exists, so we shouldn't need a replace :shrug: 
https://github.com/sourcegraph/conc
Can we drop this completely?

---

I just did. This was me cargo-culting the prior lib redirect, because I misread the conc one. Removed.